site/content/docs/5.2/examples/form/form.js

```diff
@@ -0,0 +1,24 @@
+(() => {
+  'use strict'
+
+  // Tooltip
+  const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
+  const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new boosted.Tooltip(tooltipTriggerEl)) // eslint-disable-line no-unused-vars, no-undef
+
+  // Fetch all the forms we want to apply custom Bootstrap validation styles to
+  const forms = document.querySelectorAll('.needs-validation')
+  // Loop over them and prevent submission
+  Array.from(forms).forEach(form => {
+    form.addEventListener('submit', event => {
+      if (!form.checkValidity()) {
+        event.preventDefault()
+        event.stopPropagation()
+        // Focus on first error
+        const invalidItems = form.querySelectorAll(':invalid')
```

Comment:
In order to make it longer I can add the whole page instead of just the form itself. But it will not be a form example anymore but an example of a page containing a form. What do you think @julien-deramond?

Reply:
I would go for a page based on [my other comment in this same PR.](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1559#discussion_r1013635533).